### DISCUSSION TITLE
May I ask for your advice

### DISCUSSION THREAD

**Discussion by qiuliunian** [rvdbreemen/OTGW-firmware#251] [Q&A]
Download [.bin files](https://github.com/rvdbreemen/OTGW-firmware/releases/latest) and rename the .bin files to ino.bin and fs.bin
I found only one BIN file

**Comment by DaveDavenport** [comment]
there are two on the page you link:
https://github.com/rvdbreemen/OTGW-firmware/releases/download/v0.10.3/OTGW-firmware.0.10.3+e334c42.littlefs.bin
and
https://github.com/rvdbreemen/OTGW-firmware/releases/download/v0.10.3/OTGW-firmware-0.10.3+e334c42.ino.bin
